include/openspace/engine/configuration.h

```diff
@@ -45,6 +45,7 @@ struct Configuration {
 
     std::string windowConfiguration = "${CONFIG}/single.xml";
     std::string asset;
+    std::string profileOutPrefixName;
```

Comment:
This variable is only set once in OpenSpaceEngine::initialize and then read in OpenSpaceEngine::loadInitAssetSection.   The second function is never called, meaning that the stored value is never read, meaning that the variable can be removed entirely

Reply:
Also the Configuration::usingProfile variable is not necessary any more either

Reply:
Removed `profileOutPrefixName` and `usingProfile` variables.